docs/source/integrations/spark.rst

```diff
@@ -13,8 +19,10 @@ Make sure to have at least BentoML 1.0.13 and Spark version 3.3.0 available in y
 
 .. code-block:: bash
 
-	$ pip install -U "bentoml>=1.0.13"
+    $ pip install -U "bentoml>=1.0.13"
 
+BentoML and your service's must also be installed in the Spark cluster. Most likely, the service
```

Comment:
Thanks for adding this. Should we mention that users will need to install the dependencies in addition to bentoml?

Reply:
Ah, I think I didn't finish that sentence.